src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -40,62 +40,92 @@ class SeekBar extends Slider {
    */
   constructor(player, options) {
     super(player, options);
-    this.setEventHandlers_();
+    this.update_ = Fn.bind(this, this.update);
```

Comment:
Was kind of silly to rebind these in a helper, and we need them in both `doEvent` functions.